examples/client.go

```diff
@@ -644,12 +644,28 @@ func main() {
 		fmt.Println("GetJobs...role: ", role)
 		_, result, err := r.GetJobs(role)
 		if err != nil {
-			fmt.Print("error: %+v\n", err.Error())
+			fmt.Printf("error: %+v\n", err.Error())
 			os.Exit(1)
 		}
 		fmt.Println("map size: ", len(result.Configs))
 		fmt.Println(result.String())
 
+	case "snapshot":
+		fmt.Println("Forcing scheduler to write snapshot to mesos replicated log")
+		err := r.Snapshot()
```

Comment:
for example code, we can just `log.Fatal(err)`  which is `fmt.Printf` + `os.Exit` with timestamp